routes.js

```diff
@@ -57,8 +57,8 @@ router.post('/event-question/response', auth.authUser, (req, res) => eventQuesti
 router.delete('/event-question/response/:eventQuestionId/:responseId', auth.authAdmin, (req, res) => eventQuestionController.deleteResponse(req, res));
 
 // ResponseType routes
-router.get('/response-type', auth, (req, res) => responseTypeController.getAll(req, res));
-router.get('/response-type/:value', auth,  (req, res) => responseTypeController.getByValue(req,res));
+router.get('/response-type', auth.authAdmin, (req, res) => responseTypeController.getAll(req, res));
```

Comment:
@SavoX2 changed these response type auths to auth.authUser so you two should come to an agreement which one to use.

Reply:
@SavoX2 is response-type route for all users or only admin specific?